hc/accounts/tests/test_login.py

```diff
@@ -17,20 +17,26 @@ def test_it_sends_link(self):
         session.save()
 
         form = {"email": "alice@example.org"}
+        resp = self.client.post("/accounts/login/", form)
+        # self.assertEqual(resp.status_code, 302)
+        self.assertRedirects(resp, "/accounts/login_link_sent/")
 
-        r = self.client.post("/accounts/login/", form)
-        assert r.status_code == 302
-
+        # import ipdb;ipdb.set_trace()
         ### Assert that a user was created
-
+        user = User.objects.get(email=form["email"])
+        #created_user = User.objects.find(id=user.id)
```

Comment:
It is commented out because it is not in use.
It has been removed in new commit.